code/datums/wires/wires.dm

```diff
@@ -283,6 +283,47 @@
 /datum/wires/proc/get_wire(color)
 	return colors[color]
 
+/**
+ * Get the name of the wire passed in.
+ * If wires can't be read, may either return something scrambled or nothing at all.
+ * A null return here means that there should be no name displayed at all.
+ *
+ * Arguments:
+ * * wire - a wire define.
+ */
+/datum/wires/proc/get_wire_name(mob/user, wire)
+	if(user.can_admin_interact())
+		// admins always get a free pass
+		return wire
+
+	var/can_probably_see_wires = FALSE
+	var/obj/item/held_item = user.get_active_hand()
+	if(istype(held_item) && HAS_TRAIT(held_item, TRAIT_SHOW_WIRE_INFO))
```

Comment:
How does this cope with things held in the off-hand after initially opening the wire interface with them? (open, swap hands, wire interact) does it remove the viewing of wires? or is it always there once the interface is opened?

Reply:
Good catch! It doesn't.
![image](https://user-images.githubusercontent.com/89928798/225419426-d400c308-bc2c-49f0-83ec-f046e02d6905.png)
I'll make it check offhand as well.